cdk_emqx_cluster/cdk_emqx_cluster_stack.py

```diff
@@ -451,7 +458,10 @@ def setup_monitoring(self, targets):
         c_grafana = task.add_container('grafana',
                                        essential=True,
                                        image=ecs.ContainerImage.from_registry(
-                                           'ghcr.io/emqx/perf-test-grafana:master'),
+                                           'ghcr.io/k32/sysmon-grafana:0.1.1'),
+                                       environment={
+                                           'POSTGRES_PASS': self.postgresPass
```

Comment:
Sometimes it's `POSTGRES_PASS` and sometimes `POSTGRES_PASSWORD`, is it different parameter naming in different containers?

Reply:
Yes, correct.